core/sail/extensible-store/src/test/java/org/eclipse/rdf4j/sail/extensiblestoreimpl/TransactionIsolationAndWalTests.java

```diff
@@ -26,7 +26,6 @@
 	 * transaction should see either nothing added or everything added. Nothing in between.
 	 */
 	@Test
-	@Ignore
 	public void testReadCommittedLargeTransaction() throws InterruptedException {
```

Comment:
I don't understand why you need a custom test for this. We have a whole testsuite full of isolation compliance tests that you could reuse for this purpose. Why not simply extend  `SailIsolationLevelTest`?

Reply:
(not saying you can't have more tests than what the testsuite already offers of course, but this particular thing you're testing here seems to me to be well-covered).

Reply:
This test interacts with the underlying datastructure to consistently reproduce the isolation issue. The isolation tests we currently have do not seem to cover this issue here or any write-ahead-logging issues.

Reply:
I will look into adding this to the suite instead.